src/credentials/FabricTable.cpp

```diff
@@ -2180,4 +2181,72 @@ CHIP_ERROR FabricTable::SetShouldAdvertiseIdentity(FabricIndex fabricIndex, Adve
     return CHIP_NO_ERROR;
 }
 
+CHIP_ERROR FabricTable::FetchVIDVerificationStatement(FabricIndex fabricIndex, MutableByteSpan & outVIDVerificationStatement) const
+{
+    VerifyOrReturnError(mOpCertStore != nullptr, CHIP_ERROR_INCORRECT_STATE);
+    VerifyOrReturnError(outVIDVerificationStatement.size() >= kVendorIdVerificationStatementV1Size, CHIP_ERROR_BUFFER_TOO_SMALL);
+
+    // TODO(#38308): Add VIDVerificationStatement loading support.
+    outVIDVerificationStatement.reduce_size(0);
+    return CHIP_NO_ERROR;
```

Comment:
would it make sense to have CHIP_ERROR_NOT_IMPLEMENTED for these TODOs? or do we want it to pass for now (in that case maybe a comment with the TODO saying we say succes on purpose may help.

Reply:
We want it to pass for now as if not there since the command to set them is unimplemented. Will update comment in a follow-up.